addons/mail/static/src/components/activity/activity.xml

```diff
@@ -47,51 +47,57 @@
+                                    </span>
+                                </div>
+                                <div class="d-table-row" t-if="activityView.activity.creator">
+                                    <b class="d-table-cell text-right py-1 px-4">Created</b>
+                                    <span class="o_Activity_detailsCreation d-table-cell py-1 pr-4">
+                                        <t t-esc="activityView.formattedCreateDatetime"/>, by&#160;
+                                        <img class="o_Activity_detailsUserAvatar o_Activity_detailsCreatorAvatar rounded-circle" t-attf-src="/web/image/res.users/{{ activityView.activity.creator.id }}/avatar_128" t-att-title="activityView.activity.creator.nameOrDisplayName" t-att-alt="activityView.activity.creator.nameOrDisplayName"/>
+                                        <b class="o_Activity_detailsCreator">
                                             <t t-esc="activityView.activity.creator.nameOrDisplayName"/>
+                                        </b>
+                                    </span>
+                                </div>
+                                <div class="d-table-row" t-if="activityView.activity.assignee">
+                                    <b class="d-table-cell text-right py-1 px-4">Assigned to</b>
+                                    <span class="o_Activity_detailsAssignation d-table-cell py-1 pr-4">
+                                        <img class="o_Activity_detailsUserAvatar o_Activity_detailsAssignationUserAvatar rounded-circle" t-attf-src="/web/image/res.users/{{ activityView.activity.assignee.id }}/avatar_128" t-att-title="activityView.activity.assignee.nameOrDisplayName" t-att-alt="activityView.activity.assignee.nameOrDisplayName"/>
+                                        <b t-esc="activityView.activity.assignee.nameOrDisplayName"/>
+                                    </span>
+                                </div>
+                                <div class="d-table-row">
+                                    <b class="d-table-cell text-right py-1 px-4">Due on</b>
+                                    <span class="o_Activity_detailsDueDate d-table-cell py-1 pr-4">
+                                        <span class="o_Activity_deadlineDateText"
+                                            t-att-class="{
+                                                'o-default': activityView.activity.state === 'default',
+                                                'o-overdue': activityView.activity.state === 'overdue',
+                                                'o-planned': activityView.activity.state === 'planned',
+                                                'o-today': activityView.activity.state === 'today',
+                                            }"
+                                        >
+                                            <t t-esc="activityView.formattedDeadlineDate"/>
```

Comment:
Why is github marking a lot of these lines as changed?

I don't see any change in them, but maybe it will show something different in editor?

Reply:
Ok I get it, there is a partial indentation change -> the blocks that had `t-if` don't have indent change because you re-used the `t-if` level to add the `div`, but those that didn't have the condition now have an extra div.

Reply:
> but maybe it will show something different in editor?

Nope :/
Somehow "fixed" ( I just reverted the line and amended the commit, not pushed yet)